packages/react-components/components/select/src/select.jsx

```diff
@@ -69,6 +74,7 @@ export function PureSelect(props) {
             options={renderOptions()}
             selection={!inline}
             inline={inline}
+            multiple={multiple}
```

Comment:
You dont need to add `multiple` here since you don't use it, it would be provided to the select component through `...rest`.